asm-utils/src/data_parser.rs

```diff
@@ -109,6 +169,24 @@ fn extract_data_value<R: Register, F: FunctionOpKind>(
             data.push(0);
             vec![DataValue::Direct(data)]
         }
+        (".dword" | ".half" | ".hword" | ".short", data) => {
+            let len = match directive {
+                ".dword" => 8,
+                _ => 2,
+            };
```

Comment:
Since this is already handling different sizes, can't we also support `.byte` here directly? The rest of the code looks similar

Reply:
Indeed.